docs/developing.rst

```diff
@@ -0,0 +1,58 @@
+Developing
+==========
+
+Install the requirements
+
+::
+
+    $ pip install -r requirements.txt
+
+(note that lxml, used in the tests, requires that libxslt and libxml2 be
+installed on the system)
+
+Clone the git repository:
+
+::
+
+    $ git clone https://github.com/unt-libraries/django-major-event-log
+    $ cd django-major-event-log
+
+Initialize the database
+
+::
+
+    $ python manage.py migrate
+
+Create a superuser so you can create test events
+
+::
+
+    $ python manage.py createsuperuser
+
+Start the development server
+
+::
+
+    $ python manage.py runserver
+
+(press Ctrl-c to stop the development server)
+
+You should now be able to access both the admin portion of the app at
```

Comment:
Nitpick: I think we could take out "both," since it doesn't really go with "or," and I don't think we need a comma before "or." Ok, I'm done with comments for the upcoming commit :).